### DISCUSSION TITLE
New Home screen layout mock up.

### DISCUSSION THREAD

**Discussion by tiniwings** [iSoron/uhabits#1952] [Feature Requests]
Hi, 
Can you please check the attached mockup screenshots for Home/main screen page? 

The current layout provides only limited information on first glance. 

In Smaple picture2, you can see the Icon, Title, description and Done checkmark for that day. 

![Mockup](https://github.com/iSoron/uhabits/assets/37637608/1705a1e3-d2a4-40af-82ee-58df060b90b3)
![Sample2](https://github.com/iSoron/uhabits/assets/37637608/37d096ba-8e96-4eab-a019-a258aaa9c6fd)

Applink: [removed link to non open source app]

Thank you.

**Comment by sevenate** [comment]
I was considered suggesting exactly this, but you beat me 😆.

Currently I had to use both apps as Habits provides a nice compact view of a long list of habits (11+), but with only "up to 4 days history at a glance" and referenced HabitKit gives me 27 weeks (!) of history at a glance (without digging into every habit individually), but only 4 habits fit on screen without scrolling, which is also a bummer in a way.

I wish any of the apps could let me choose the "density" of information they provide on the main page, i.e. how many days of history and how detailed each day should look like (e.g. show the counts number or just filled square with intensity proportional to how many counts was completed so far in that day).
